xc/device.ml

```diff
@@ -1650,7 +1650,8 @@ let prepend_wrapper_args domid args =
  * exception is raised *)
 let init_daemon ~task ~path ~args ~name ~domid ~xs ~ready_path ?ready_val ~timeout ~cancel _ =
 	debug "Starting daemon: %s with args [%s]" path (String.concat "; " args);
-	let pid = Forkhelpers.safe_close_and_exec None None None [] path args in
+	let syslog_stdout = Forkhelpers.Syslog_WithKey "QEMU_start" in
```

Comment:
I think we should use the basename of the executable for the syslog key. In this case, `path`.